modules/www/manifests/init.pp

```diff
@@ -64,6 +57,13 @@
   }
 
   if $competitor_services {
+    file { '/etc/sr':
+      ensure => directory,
+      owner => 'root',
+      group => 'root',
+      mode => '755',
+    }
+
```

Comment:
I'm not sure having this in a conditional is a good idea -- puppet only allows a single declaration of each target thing, meaning that if other things turn out to also need this we'd be creating a coupling between them and needing to set `$competitor_services` where no such dependency might actually want to exist.

```suggestion
  file { '/etc/sr':
    ensure => directory,
    owner => 'root',
    group => 'root',
    mode => '755',
  }

  if $competitor_services {
```
Apologies if putting it in here was what you though I meant in https://github.com/srobo/server-puppet/pull/1#discussion_r550905491